vital/types/local_geo_cs.cxx

```diff
@@ -35,7 +35,7 @@ ::set_origin(const geo_point& origin)
   vector_3d lon_lat_alt = origin.location( SRID::lat_lon_WGS84 );
   auto zone = utm_ups_zone( lon_lat_alt );
   int crs = (zone.north ? SRID::UTM_WGS84_north : SRID::UTM_WGS84_south) + zone.number;
-  geo_origin_ = geo_point(origin.location(crs), crs);
```

Comment:
passing `crs` is required for this to work in the general case.  You must first convert the origin from whatever space it was in to this new `crs`, then you construction a new `geo_point` where this `crs` is the default representation.